### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by swenson**
By running:

```sh
go list -u -m -json all | jq -r 'select(.Indirect != true and .Update != null) | .Path+"@"+.Update.Version' | xargs -L1 go get
go mod tidy
```

I also updated for Go 1.19 and fixed integration tests for ARM64 processors.

Packages updated:

|             MODULE              |              VERSION               | NEW VERSION | DIRECT | VALID TIMESTAMPS |
|---------------------------------|------------------------------------|-------------|--------|------------------|
| github.com/hashicorp/go-hclog   | v1.1.0                             | v1.3.1      | true   | true             |
| github.com/hashicorp/go-uuid    | v1.0.2                             | v1.0.3      | true   | true             |
| github.com/hashicorp/go-version | v1.2.0                             | v1.6.0      | true   | true             |
| github.com/hashicorp/vault/api  | v1.5.0                             | v1.8.2      | true   | true             |
| github.com/hashicorp/vault/sdk  | v0.5.3                             | v0.6.1      | true   | true             |
| k8s.io/api                      | v0.0.0-20190409092523-d687e77c8ae9 | v0.25.3     | true   | true             |
| k8s.io/apimachinery             | v0.22.2                            | v0.25.3     | true   | true             |

I ran the integration tests locally, and they passed.

I also checked that the Vault tests pass after updating this plugin (and the k8s secrets plugin): https://github.com/hashicorp/vault/commit/ce44aac88b6852b711fdc18b8ca39ee5f97b2d69

# Related Issues/Pull Requests
- [ ] https://github.com/hashicorp/vault-plugin-secrets-kubernetes/pull/15

# Contributor Checklist
- [x] Add relevant docs to upstream Vault repository, or sufficient reasoning why docs won’t be added yet (N/A)
- [x] Add output for any tests not ran in CI to the PR description (eg, acceptance tests)
- [x] Backwards compatible

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: 1.19.1
+          go-version: 1.19.3
```
Comment (tvoran) [inline .github/workflows/tests.yaml:18]: Looks like go-version is being set on line 51 as well.

**Review by tvoran — APPROVED** [review]

**Review by benashz — APPROVED** [review]
LGTM!

**Review by tomhjp — APPROVED** [review]
LGTM - given the timing in the release cycle, it might be a good time to add a changelog to this repo too? But I don't mind if you just want to land this as-is

**Comment by swenson** [comment]
Cool, I can add a changelog.

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (swenson) [reply .github/workflows/tests.yaml:18]: :+1:

**Comment by swenson** [comment]
Thanks!

**Pull request merged**